src/pveclib/vec_common_ppc.h

```diff
@@ -230,16 +230,17 @@ typedef __vector __bool int vb32_t;
 typedef __vector __bool long long vb64_t;
 
 /* did not get vector __int128 until GCC4.8.  */
-#if (__GNUC__ > 4) || (__GNUC__ == 4 && __GNUC_MINOR__ >= 8)
+//#if (__GNUC__ > 4) || (__GNUC__ == 4 && __GNUC_MINOR__ >= 8)
```

Comment:
should we get rid of this, since it's now covered in the `configure` processing?

Reply:
I though did already. Anyway its gone now.